nestkernel/archiving_node.h

```diff
@@ -147,6 +207,28 @@ class Archiving_Node : public Node
 
   // spiking history needed by stdp synapses
   std::deque< histentry > history_;
+
+  /*
+   * Structural plasticity
+   */
+
+  // Time of the last update of the Calcium concentration in ms
```

Comment:
This comment is minor: I suggest that you make all comments describing the private state variables  (Ca_t_, Ca_minus_, etc.) to conform to the doxygen format so that they show up in the documentstion.